feat: template crd (#4302)
kubeshop/testkube: docs/docs/cli/testkube_delete_template.md

## testkube delete template

Delete template

### Synopsis

> Delete template, pass template name which should be deleted

Reviewer: ```suggestion
Delete a template and pass the template name to be deleted.
```
